packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -692,20 +692,22 @@ describe('Ingest Service Tests', () => {
   })
 
   describe('getLatestTelemetryForDevices', () => {
+    const TEST_TELEMETRY_A = [TEST_TELEMETRY_A1, TEST_TELEMETRY_A2]
+    const TEST_TELEMETRY_B = [TEST_TELEMETRY_B1, TEST_TELEMETRY_B2]
+
     beforeEach(async () => {
-      await IngestRepository.createTelemetries([TEST_TELEMETRY_A1, TEST_TELEMETRY_B1])
-      await IngestRepository.createTelemetries([TEST_TELEMETRY_A2, TEST_TELEMETRY_B2])
+      await IngestRepository.createTelemetries([...TEST_TELEMETRY_A, ...TEST_TELEMETRY_B])
     })
 
     it('loads last telemetries per device', async () => {
-      const telemetries = await IngestServiceClient.getLatestTelemetryForDevices([
-        TEST_TELEMETRY_A1.device_id,
-        TEST_TELEMETRY_B1.device_id
-      ])
-      const timestamps = telemetries.map(({ timestamp }) => timestamp)
-      expect(timestamps[0]).toStrictEqual(testTimestamp + 1000)
-      expect(timestamps[1]).toStrictEqual(testTimestamp + 1000)
-      expect(telemetries?.length).toStrictEqual(2)
+      const device_ids = [DEVICE_UUID_A, DEVICE_UUID_B]
+      const telemetries = await IngestServiceClient.getLatestTelemetryForDevices(device_ids)
+      expect(telemetries).toHaveLength(device_ids.length)
+      if (telemetries.length === device_ids.length) {
```

Comment:
Why this if?

Reply:
Because the test will already have failed on the previous line and destructuring data further is pointless.

Reply:
I don't understand what impact the destructuring would have if the test stops after the failure.

Reply:
Yeah, I think by the time we get here, this `if` will *have* to be true, so it's redundant to check again

Reply:
I thought I had run into cases before where `Jest` would attempt to execute all matchers in an `it`. That doesn't seem to be the case here so I removed the `if`.